manifest

```diff
@@ -40,16 +40,16 @@ export PACKAGES="\
 	networkmanager \
 	pipewire \
 	lib32-pipewire \
+	pipewire-alsa \
+	pipewire-pulse \
+	pipewire-jack \
```

Comment:
I don't think we need jack.

Reply:
I've seen some packages depending on jack, but if you see it interfering, we can drop it.